survey/references/questionnaire_list_en.txt

```diff
@@ -60,8 +60,8 @@ For babies under the age of 1, write "0"
 
 Sex at birth
 Sex assigned at birth or that which appears on legal documents.
-- Man
 - Woman
+- Man
```

Comment:
Female/Male ou Women/Man? À vérifier

Reply:
Ceci est généré par le générateur, si c'est là, c'est que c'est Woman/Man

Reply:
faut vérifier le générateur via un autre issue

Reply:
> faut vérifier le générateur via un autre issue

Pourquoi? C'est quoi le problème que tu vois potentiellement? C'est man Woman/man qu'on doit avoir? On veut avoir Femme/Homme en français et Female/Male en anglais?

Reply:
Selon l'AI de Google: In questionnaires, "woman/man" is generally preferred over "female/male" when referring to gender identity, as "female/male" is often associated with biological sex.

Ici on veut le sexe à la naissance, donc ça devrait être female/male